packages/nx/src/nx-init/add-nx-to-npm-repo.ts

```diff
@@ -48,8 +48,9 @@ export async function addNxToNpmRepo() {
         {
           type: 'multiselect',
           name: 'cacheableOperations',
-          message:
-            'Which of the following scripts are cacheable? (Produce the same output given the same input, e.g. build, test and lint usually are, serve and start are not)',
+          message: `Which of the following scripts are cacheable? (Produce the same output given the same input, e.g. build, test and lint usually are, serve and start are not)
+            You can use spacebar to select one or more scripts.
```

Comment:
Isn't this going to show up indented? Can you add a screenshot of what this looks like?